lvx backend/config.env

```diff
@@ -0,0 +1,4 @@
+NODE_ENV=development
```

Comment:
You have uploaded `.env` on github it should not be here, you need to keep it private, it contains sensitive details.